### PR TITLE
Fixed a documentation of geo fields

### PR DESCRIPTION
**Pull request by prog8**
As I wrote in JIRA it is not possible to detect geo fields in ES. The only geofield we support in our default mappings is called geoip.

### CODE REVIEW

File: docs/logs/field-types.md [hunk 1]
```diff
@@ -97,17 +97,18 @@ can be useful on dashboards presenting maps, for example.
...
       "message": "Hello from Sematext!"
     }'
 
 
 **Note:**
 
-If you want to use this field type you can send documents with a field
-in a format presented above. Data in such a format will be
-automatically detected as geo-point. Alternatively, you can use Field
+If you want to use this field type you can send documents with a field name and
```
Comment (otisg) [inline docs/logs/field-types.md]: documents => events

File: docs/logs/field-types.md [hunk 2]
```diff
@@ -97,17 +97,18 @@ can be useful on dashboards presenting maps, for example.
...
     }'
 
 
 **Note:**
 
-If you want to use this field type you can send documents with a field
-in a format presented above. Data in such a format will be
-automatically detected as geo-point. Alternatively, you can use Field
+If you want to use this field type you can send events with a field name and
+data in a format presented above. Alternatively, you can use Field
```
Comment (adnanrahic) [inline docs/logs/field-types.md]: you can use Field => you can use *__the__* Field

File: docs/logs/field-types.md [hunk 3]
```diff
@@ -97,17 +97,18 @@ can be useful on dashboards presenting maps, for example.
...
 Editor to manually create a field of geo-point type.
```
Comment (adnanrahic) [inline docs/logs/field-types.md]: Editor to manually create a field of geo-point type.
=>
Editor to manually create a field of _**the**_ geo-point type.

File: docs/logs/field-types.md (hunk 2 above)
Comment (prog8) [reply docs/logs/field-types.md]: Are you sure that we should put the before "Field Editor"?

Comment (adnanrahic) [reply docs/logs/field-types.md]: Yeah.

Comment (prog8) [reply docs/logs/field-types.md]: We don't use "the" before Field Editor anywhere else, @otisg should be adjust all the places to have "the"?

Comment (otisg) [reply docs/logs/field-types.md]: Either way

**Pull request merged**